src/test/kotlin/no/nav/familie/tilbake/service/dokumentbestilling/varsel/manuelt/ManueltVarselBrevTjenesteTest.kt

```diff
@@ -0,0 +1,176 @@
+    private lateinit var manueltVarselBrevTjeneste: ManueltVarselBrevTjeneste
+    private var behandling = Testdata.behandling
+    private var fagsak = Testdata.fagsak
+
+    @BeforeEach
+    fun setup() {
+        manueltVarselBrevTjeneste = ManueltVarselBrevTjeneste(behandlingRepository,
+                                                              fagsakRepository,
+                                                              mockEksternDataForBrevTjeneste,
+                                                              mockPdfBrevTjeneste,
+                                                              mockFeilutbetalingService)
+
+        every { mockFeilutbetalingService.hentFaktaomfeilutbetaling(any()) }
+                .returns(lagFeilutbetalingFakta())
+        val personinfo = PersonInfo("DUMMY_FØDSELSNUMMER", LocalDate.now(), "Fiona")
+        val ident: String = Testdata.fagsak.bruker.ident
+        every { mockEksternDataForBrevTjeneste.hentPerson(ident, any()) }.returns(personinfo)
+        every {
+            mockEksternDataForBrevTjeneste.hentAdresse(any(), any(), any(), any())
+        }.returns(Adresseinfo("Test", "12345678901"))
+
+        fagsak = fagsakRepository.insert(fagsak)
+        behandling = behandlingRepository.insert(behandling)
+    }
+
+    @Test
+    fun `skal sende manuelt varselbrev`() {
+        manueltVarselBrevTjeneste.sendManueltVarselBrev(behandling.id, varselTekst, BrevMottaker.BRUKER)
+        verify {
+            mockPdfBrevTjeneste.sendBrev(eq(behandling),
+                                         eq(fagsak),
+                                         eq(Brevtype.VARSEL),
+                                         any(),
+                                         eq(9000L),
+                                         any())
+        }
+    }
+
+    @Test
+    fun `skal sende korrigert varselbrev`() {
+        excludeRecords { mockPdfBrevTjeneste.sendBrev(eq(behandling), eq(fagsak), eq(Brevtype.VARSEL), any(), any(), any()) }
+        //arrange
+        manueltVarselBrevTjeneste.sendManueltVarselBrev(behandling.id, varselTekst, BrevMottaker.BRUKER)
+        val behandlingCopy = behandling.copy(varsler = setOf(Varsel(varseltekst = varselTekst,
+                                                                    varselbeløp = 100L)))
+        val behandling = behandlingRepository.update(behandlingCopy)
+
+        //act
+        manueltVarselBrevTjeneste.sendKorrigertVarselBrev(behandling.id, korrigertVarselTekst, BrevMottaker.BRUKER)
+
+        //assert
+        verify {
+            mockPdfBrevTjeneste.sendBrev(eq(behandling),
+                                         eq(fagsak),
+                                         eq(Brevtype.KORRIGERT_VARSEL),
+                                         any(),
+                                         eq(9000L),
+                                         any())
+        }
+    }
+
+    @Test
+    fun `skal sende korrigert varselbrev med verge`() {
```

Comment:
bruker du ikke metode navn i begynnelsen?

Reply:
Jeg foreslo på stand up for et par uker siden at vi ikke skulle gjenta samme kommentaren på 50 steder med identisk setting, fordi PRen blir uoversiktlig og det øker sjansen for at kommentarer blir oversett.